src/app/routes/index.tsx

```diff
@@ -334,6 +323,10 @@ const router = createHashRouter([
           </AuthGuard>
         ),
       },
+      {
+        path: 'send-nft/:id',
+        element: <SendNft />,
+      },
```

Comment:
🤔 is this new route ledger specific?

Reply:
Currently yes, I added this route to the route array which uses a different wrapper so that it looks like this in a full screen, to be consistent with other ledger-related screens:
<img width="1912" alt="Monosnap Xverse Wallet 2023-09-01 17-49-38" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/d321210a-1fe1-4eb6-9902-59505799e50b">

Here's how the same component looks for regular accounts in a full screen:
<img width="1907" alt="Monosnap Xverse Wallet 2023-09-01 17-49-19" src="https://github.com/secretkeylabs/xverse-web-extension/assets/36603049/d1f72e2c-cf0d-42c1-9607-1e75ef71f3bc">

I think it should be updated for the regular accounts as well, but I didn't want to break something as this PR already touches a lot of things, should we create a separate task to improve the view for the regular accounts? (probably remove the old route and use this one for both ledger & regular accounts)

Reply:
we have to update it as part of Collectibles UI project I believe, so we can do it along with that 👍